packages/amplify-provider-awscloudformation/src/utils/admin-helpers.ts

```diff
@@ -27,20 +27,71 @@ export async function isAmplifyAdminApp(appId: string): Promise<{ isAdminApp: bo
   return { isAdminApp: !!appState.appId, region: appState.region };
 }
 
+export async function getTempCredsWithAdminTokens(appId: string, print: $TSContext['print']): Promise<AwsSdkConfig> {
```

Comment:
I don't see this function called anywhere? Is that coming in a future PR or am I missing something?

Reply:
It gets imported and called in `configuration-manager.ts`